pkg/cli/cmd/radinit/aws.go

```diff
@@ -27,15 +28,15 @@ import (
 )
 
 const (
-	// QueryRegion is the region used for querying AWS before the user selects a region.
-	QueryRegion = "us-east-1"
-
 	selectAWSRegionPrompt                 = "Select the region you would like to deploy AWS resources to:"
 	enterAWSIAMAcessKeyIDPrompt           = "Enter the IAM access key id:"
 	enterAWSIAMAcessKeyIDPlaceholder      = "Enter IAM access key id..."
 	enterAWSIAMSecretAccessKeyPrompt      = "Enter your IAM Secret Access Key:"
 	enterAWSIAMSecretAccessKeyPlaceholder = "Enter IAM secret access key..."
 	errNotEmptyTemplate                   = "%s cannot be empty"
+	confirmAWSAccountIDPromptFmt          = "Use account id '%v'?"
+	enterAWSAccountIDPrompt               = "Enter the account ID you want to use:"
+	enterAWSAccountIDPlaceholder          = "Enter the account ID you want to use.."
```

Comment:
Could you please check that the format/phrasing is consistent with rest of the prompts?

```suggestion
	enterAWSAccountIDPrompt               = "Enter the account ID:"
	enterAWSAccountIDPlaceholder          = "Enter the account ID you want to use..."
```